Tests/UnitTests/Plugins/Sycl/Seeding/SeedfinderSyclTest.cpp

```diff
@@ -122,6 +126,7 @@ auto setupSpacePointGridConfig(
 
 auto main(int argc, char** argv) -> int {
   auto start_prep = std::chrono::system_clock::now();
+  vecmem::sycl::shared_memory_resource* resource = nullptr;
```

Comment:
:stuck_out_tongue: I was expecting something of a bit more elaborate issue. :smile:

It was correct to receive the `vecmem::memory_resource` pointer in `Acts::Sycl::Seedfinder` as you did. That part you did as I imagined.

But then, you need to give a valid pointer to the seedfinder. :wink: The most easily by creating your `vecmem::sycl::shared_memory_resource` on the stack. Like:

```c++
vecmem::sycl::shared_memory_resource resource;
```

This will create an actual object in memory, not just a 64-bit empty `nullptr`. Then, and I guess this is what confused you, you can give this object's pointer to the seedfinder constructor like:

```c++
  Acts::Sycl::Seedfinder<SpacePoint> syclSeedfinder(
      config, deviceAtlasCuts,
      &resource,
      Acts::Sycl::QueueWrapper(
      ...
```

I.e. you can use `&` to give a pointer to this real object to the seedfinder.

Give that a try. :wink:

Reply:
Ooh, obviously, that makes more sense.